packages/component-library/src/HorizontalBarChart/HorizontalBarChart.js

```diff
@@ -50,6 +52,32 @@ const HorizontalBarChart = ({
 
   const minValue = Math.min(0, ...data.map(d => d[dataValue]));
 
+  const categoricalColors = dataLength => {
```

Comment:
This seems like it could be a useful helper for other stories, could it go in a utils folder?

Reply:
I totally agree! Will do.